cosmos/shido_9008.json

```diff
@@ -0,0 +1,60 @@
+{
+  "rpc": "https://tendermint.shidoscan.com",
+  "rest": "https://swagger.shidoscan.com",
+  "chainId": "shido_9008-1",
+  "chainName": "Shido",
+  "evm": {
+    "chainId": 9008,
+    "rpc": "https://rpc-nodes.shidoscan.com"
+  },
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/shido_9008/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "SHIDO",
+    "coinMinimalDenom": "shido",
+    "coinDecimals": 18,
+    "coinGeckoId": "shido-2",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/shido_9008/shido.png"
+  },
+  "nodeProvider":{
+    "name":"Shido",
+    "email":"info@shido.finance",
+    "website":"https://shido.io/"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/shido",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/shido",
```

Comment:
The page isn't supported by Keplr Dashboard dynamically.
You can remove these two properties from the change.